build.gradle.kts

```diff
@@ -13,16 +13,20 @@ repositories {
 
 dependencies {
     implementation("org.glassfish.grizzly:grizzly-http-server:2.4.4")
-    implementation("org.json:json:20200518")
+    implementation("org.json:json:20220320")
     implementation("org.glassfish.jersey.core:jersey-server:2.34")
     implementation("org.glassfish.jersey.media:jersey-media-json-jackson:2.34")
     implementation("org.glassfish.jersey.inject:jersey-hk2:2.34")
     implementation("org.glassfish.jersey.containers:jersey-container-grizzly2-http:2.34")
     implementation("com.fasterxml.jackson.module:jackson-module-kotlin:2.12.1")
     implementation("org.jetbrains.kotlinx:kotlinx-serialization-json:1.3.0")
-
+    implementation("com.google.code.gson:gson:2.8.9")
+    implementation("org.jetbrains.kotlinx:kotlinx-serialization-protobuf:1.3.0")
+    implementation("org.jetbrains.kotlinx:kotlinx-serialization-core:1.3.0")
+    implementation("org.mongodb:mongodb-driver-sync:4.3.1")
```

Comment:
usemongo driver client version 4.7.0 as our applications are running on this latest version

Reply:
done